### DISCUSSION TITLE
Looking for Logos

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Gordian-Developer-Community#31] [General & Announcements]
Hey Folks,

I'm creating a graphic for the front page of this repo, and to feature in any social media posts. I'd love to fill it up with all the people who are talking here. If you'd like to be included, please post a transparent version of your company and/or wallet logo (PNG, AI, or EPS), and I'll put them all together after Thanksgiving.

**Comment by wolfmcnally** [comment]
![WolfMcNally](https://user-images.githubusercontent.com/676176/100174017-7f37ff80-2e80-11eb-96c5-df976c02fb82.png)

**Comment by cuongleqq** [comment]
Hi Shannon, this is great idea!
Bitmark logo (AI version) is at https://drive.google.com/file/d/16TqbZdJc7YY2xYM2QCPCCwmmiKhd4L-Q/view?usp=sharing

**Comment by craigraw** [comment]
![sparrow](https://user-images.githubusercontent.com/862166/100200286-09c34200-2f07-11eb-98ca-67f74142e855.png)

Sparrow Wallet

**Comment by shannona** [comment]
Thanks folks. Anyone else want to jump in? I'd love to show the strength of our community.

(I'll probably create a first image tomorrow.)

**Comment by Overtorment** [comment]
https://bluewallet.io/img/logo-bluewallet.svg

**Comment by zachherbert** [comment]
Thanks! 

https://foundationdevices.com/wp-content/themes/fndn/images/fndn_logo.svg

**Comment by Fonta1n3** [comment]
Thanks, here is Fully Noded logo which is using UR airgapped QR codes. [https://github.com/Fonta1n3/FullyNoded/blob/master/Images/fn_logo.png](https://github.com/Fonta1n3/FullyNoded/blob/master/Images/fn_logo.png)

**Comment by shannona** [comment]
Thanks folks. It took me a bit, but I've created a first-cut (out on the front page).

**Comment by ChristopherA** [comment]
If you are a member of this community, we'd be glad to add your logo to this README, which also shows up as social media previous whenever anyone links to this Github community.

![](https://github.com/BlockchainCommons/Airgapped-Wallet-Community/blob/master/images/logos/airgapped-wallet-community-screen.jpg?raw=true)

-- Christopher Allen
